astroquery/alma/tests/test_alma_remote.py

```diff
@@ -149,7 +149,18 @@ def test_data_proprietary(self, alma):
         with pytest.raises(AttributeError):
             alma.is_proprietary('uid://NON/EXI/STING')
 
-    @pytest.mark.xfail(reason="Depends on PR 2438 (https://github.com/astropy/astroquery/pull/2438)")
+    def test_retrieve_data(self, temp_dir, alma):
+        """
+        Regression test for issue 2490 (the retrieval step will simply fail if
+        given a blank line, so all we're doing is testing that it runs)
+        """
+        alma.cache_location = temp_dir
```

Comment:
Is there a good reason not to use the [`temp_path` fixture](https://docs.pytest.org/en/stable/how-to/tmp_path.html) from `pytest` itself?
```suggestion
    def test_retrieve_data(self, temp_path, alma):
        """
        Regression test for issue 2490 (the retrieval step will simply fail if
        given a blank line, so all we're doing is testing that it runs)
        """
        alma.cache_location = temp_path
```

Reply:
not that I know of; I just copy-pasted another test